default.json

```
@@ -56,15 +72,19 @@
         "mcr.microsoft.com/dotnet/aspnet",
         "mcr.microsoft.com/dotnet/runtime",
         "mcr.microsoft.com/dotnet/runtime-deps",
-        "/^[mM]icrosoft\\./",
-        "/^[sS]ystem\\./"
+        "/^Azure\\.Identity/i",

```

Should it be? (similar for other regexes)
```suggestion
        "/^Azure\\.Identity$/i",
```